iidm/iidm-impl/src/main/java/com/powsybl/iidm/network/impl/extensions/ReferencePriorityImpl.java

```diff
@@ -23,7 +23,8 @@ class ReferencePriorityImpl implements ReferencePriority {
     ReferencePriorityImpl(Terminal terminal, int priority) {
         this.terminal = Objects.requireNonNull(terminal, "Terminal needs to be set for ReferencePriority extension");
         if (priority < 0) {
-            throw new PowsyblException("Priority should be zero or positive for ReferencePriority extension");
+            throw new PowsyblException(String.format("Priority (%s) of terminal should be zero or positive for ReferencePriority extension",
+                priority));
```

Comment:
Maybe we could add the id of the terminal's connectable in this message (since the `ReferencePriorities` is an extension for `Connectable`s):
```suggestion
            throw new PowsyblException(String.format("Priority (%s) of terminal (equipment %s) should be zero or positive for ReferencePriority extension",
                priority, terminal.getConnectable().getId()));
```

Reply:
done